### ISSUE TITLE
SvelteKit Frontend

### ISSUE THREAD

**Issue by Galileo-dev** [Galileo-dev/Testing-CS4442#10]
**Is your feature request related to a problem? Please describe.**
Use [SvelteKit ](https://kit.svelte.dev/) for frontend instead of basic HTML/JS 
Allow for quick prototyping and end-to-end testing through frameworks like Playwright 

**Describe the solution you'd like**
Implement a SevlteKit frontend with Tailwind CSS

**Describe alternatives you've considered**
React - Too cumbersome and too bloated
Svelte - As an alternative to the SvelteKit. after some research svelte kit seems like the best option as it scales easily. While we need routing now it's best we have a quick solution when needed. Svelte without Svelte kit is only for Single Page Applications (SPA) and doesn't feature Server-Side Rendering (SSR)

**Comment by ConorCallanan2003** [comment]
I don't think we need anything more that a single-page application though? We're not gonna get any extra marks for having a nicer UI or for using a modern JS framework.

**Comment by Galileo-dev** [comment]
It may be easier to just use something like Svelte kit to connect to the backend api. It also allows us to easily add more tests 😉 with playwright (which would get us more marks).

Also Svelte kit makes things easier not more difficult.

We don't need to do anything extra. Normal JavaScript we need to handle state and firebase OAuth is not as well supported. 

Svelte kit is very easy to use in comparison to both react and plain JavaScript.

**Comment by ConorCallanan2003** [comment]
I'll leave it up to you to decide I guess but I think the existing frontend is grand for what we're doing and we can still write playwright tests for HTML/vanilla js.
